packages/user-mgnt/src/features/system/handler.ts

```diff
@@ -324,6 +331,58 @@ export async function getSystemHandler(
   }
 }
 
+interface IGetClientIdPayload {
```

Comment:
```suggestion
interface IRefreshSYSClientSecretPayload {
```
Seems more appropriate